content/en/data_security/kubernetes.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Kubernetes Data Security
+kind: documentation
+further_reading:
+- link: "/data_security/"
+  tag: "Documentation"
+  text: "Review the main categories of data submitted to Datadog"
+---
+
+<div class="alert alert-info">This page is about the security of data sent to Datadog. If you're looking for cloud and application security products and features, see the <a href="/security/" target="_blank">Security</a> section.</div>
+
+This guide explains how to navigate permissions and security in Kubernetes when deploying and configuring the Datadog Agent, so that you can maintain both the security of your Kubernetes environment and the necessary functionality of your monitoring systems.
+
+### Permissions versus operational needs
+In the context of Kubernetes security, a balance is essential between the principle of least privilege and the smooth operation of vital components like the Datadog Agent. While Kubernetes pod security standards and CIS Benchmarks frequently emphasize minimizing permissions to avoid elevated access violations, it's necessary to acknowledge that monitoring tools like the Datadog Agent may demand certain permissions for optimal performance.
+
+## Common violations of security standards
+
+### Restricted Security Level in Kubernetes Pod Security Admission
+The Restricted pod security standard is the most strict security standard level enforced by the built in admission controller. It's intended to enforce pod security best practices at the cost of compatibility and functionality. Intended use cases include applications that process sensitive financial information and PII. 
+
+Running the Datadog agent with the restricted pod standards would prevent the agent from accessing the data required to be useful and is not recommended.
+
+### Pod running with hostPID
+A container running in the host's PID namespace can inspect processes running outside the container. If the container also has access to ptrace capabilities this can be used to escalate privileges outside of the container.
```

Comment:
```suggestion
A container running in the host's process ID (PID) namespace can inspect processes running outside the container. If the container also has access to `ptrace` capabilities, this can be used to escalate privileges outside of the container.
```